fusillade/clouddirectory.py

```diff
@@ -1607,39 +1656,21 @@ def provision_user(
         :return:
         """
         user = cls(name)
-        _creator = creator if creator else "fusillade"
+        _creator = creator if creator else None
 
         # verify parameters
         if roles:
             Role.exists(roles)
         if groups:
             Group.exists(groups)
-        if statement:
-            user._verify_statement(statement)
```

Comment:
Is there no need to verify the statement anymore?

**EDIT:** never mind, saw that it was moved to `User.create()`